collector-config.yaml

```diff
@@ -11,7 +11,7 @@ receivers:
 
 exporters:
   jaeger:
-    endpoint: jaeger:14250
+    endpoint: http://dev-otlp-lb-1255345766.us-east-1.elb.amazonaws.com:8082
```

Comment:
is this going to be replaced somehow? This needs to be dynamic, otherwise when we enable CI to test the infra it will fail I think